BUILD.gn

```diff
@@ -32,12 +32,6 @@ declare_args() {
   enable_desktop_capturer = true
   enable_run_as_node = true
   enable_osr = false
-
-  # Provide a fake location provider for mocking
-  # the geolocation responses. Disable it if you
-  # need to test with chromium's location provider.
-  # Should not be enabled for release build.
-  enable_fake_location_provider = !is_official_build
```

Comment:
Any reason this was removed ?

Reply:
yah, I was tracking another branch because originally the gn changes weren't in master yet. I'll update.